src/resources-view.h

```diff
@@ -31,6 +44,18 @@ class ResourcesView
   virtual Gtk::Widget* widget (void) = 0;
 
   virtual void set_handler (ResourcesViewCallbacks* handler) = 0;
+  
+  virtual void enable_add_resource (bool enable) = 0;
+  virtual void enable_add_resource_group (bool enable) = 0;
+  virtual void enable_up (bool enable) = 0;
+  virtual void enable_down (bool enable) = 0;
+  virtual void enable_cut (bool enable) = 0;
+  virtual void enable_copy (bool enable) = 0;
+  virtual void enable_paste (bool enable) = 0;
+  virtual void enable_delete (bool enable) = 0;
```

Comment:
Make the method names specific. i..e enable_add_resource_button. Verbose names are okay. Easier to read later on.